pkg/parse/updater.go

```diff
@@ -248,7 +248,7 @@ func (u *Updater) addWatches(ctx context.Context, gvks map[schema.GroupVersionKi
 // watching any old resources that were removed from the apply set.
 func (u *Updater) updateWatches(ctx context.Context, gvks map[schema.GroupVersionKind]struct{}, commit string) status.MultiError {
 	klog.V(1).Info("Remediator watches updating...")
-	watchErrs := u.Remediator.UpdateWatches(ctx, gvks)
+	watchErrs := u.Remediator.UpdateWatches(ctx, gvks, commit)
```

Comment:
Can the error handing below for conflicts be removed now? Are they being handled by UpdateWatches?

Reply:
Looks like no, but you could move the conflict handling down into the UpdateWatches where it checks the RESTMapper.

Reply:
Done